src/sentry/static/sentry/app/views/organizationIntegrations/IntegrationDirectorySentryAppRow.tsx

```diff
@@ -69,33 +63,26 @@ export default class SentryApplicationRow extends React.PureComponent<Props> {
   }
 
   linkToEdit() {
-    const {isOnIntegrationPage} = this.props;
+    const {app, organization} = this.props;
+
+    if (this.isInternal) {
+      return `/settings/${organization.slug}/developer-settings/${app.slug}/`;
```

Comment:
@NisanthanNanthakumar Honestly, we don't really need this if we build the redirect on `/settings/${organization.slug}/sentry-apps/${app.slug}/` for internal integrations like we talked about.

Reply:
@scefali Yeah we'll do that too, but this doesn't hurt